### PR TITLE
Add tg3 driver to support the new Tyan platform

### PR DESCRIPTION
**Pull request by ira-lb**
New machine comes with tg3 instead of e1000 and needed a kernel upgrade (tg3 was not in Fedora 25 for some reason).

New kernel is 4.16, works nice with everyone and comes with tg3.

### CODE REVIEW

**Comment by ira-lb** [comment]
@maor-lb I was asked by George to give you the final say on deploying this. Let's talk in the morning.

File: docker/build-inaugurator.dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM fedora:25
-MAINTAINER eliran@stratoscale.com
+FROM fedora:27
+MAINTAINER ops@lightbitslabs.com
```
Comment (adir-lb) [inline docker/build-inaugurator.dockerfile]: nit: the maintainer change should go into a separate commit.

Comment (ira-lb) [reply docker/build-inaugurator.dockerfile]: nitpicking indeed.

**Comment by ira-lb** [comment]
Talked to Maor, he says deployment should wait till tomorrow. in the meantime I'm merging this.

File: Makefile.build [hunk 2]
```diff
@@ -1,4 +1,4 @@
-KERNEL_VERSION=4.13.16-100.fc25.x86_64
+KERNEL_VERSION=4.16.7-200.fc27.x86_64
```
Comment (sashas-lb) [inline Makefile.build]: arent we in code freeze ?

**Comment by ira-lb** [comment]
the LightBox product is in CF. the TestOSterone product is a separate product and it's not in code freeze.

**Comment by ira-lb** [comment]
either way, this is not deployed yet.
